test/flows.spec.js

```diff
@@ -38,12 +38,18 @@ describe('Flows', function() {
     it('Allows me to: Create a plan and subscribe a customer to it', function() {
       return expect(
         Promise.join(
-          stripe.plans.create({
-            id: 'plan' + testUtils.getRandomString(),
-            amount: 1700,
-            currency: CURRENCY,
-            interval: 'month',
-            name: 'Gold Super Amazing Tier',
+          stripe.products.create({
+            name: 'product' + testUtils.getRandomString(),
+            type: 'service',
+          }).then(function(product) {
+            return stripe.plans.create({
+              id: 'plan' + testUtils.getRandomString(),
+              amount: 1700,
+              currency: CURRENCY,
+              interval: 'month',
+              nickname: 'Gold Super Amazing Tier',
+              product: product.id,
```

Comment:
You can create the plan and product in a single API request, by passing a dictionary for the [`product`](https://stripe.com/docs/api#create_plan-product) parameter. In the interest of making the test suite faster (until we move to stripe-mock 🤞 ), maybe that'd be preferable?

Reply:
Ah, excellent point! Updated.

ptal @ob-stripe